File: src/main/java/baboon/industry/item/battery/ItemBatteryCrystal.java
package baboon.industry.item.battery;

import baboon.industry.IndustryConfig;

public class ItemBatteryCrystal extends ItemBatteryBase {

    public ItemBatteryCrystal(String name, int i) {
        super(name, i,
                IndustryConfig.cfg.getInt("Energy Values.hvBatteryStorage"),
                IndustryConfig.cfg.getInt("Energy Values.hvIO"),
                IndustryConfig.cfg.getInt("Energy Values.hvIO"),
                "battery_crystal_4.png",
                "battery_crystal_3.png",
                "battery_crystal_2.png",
                "battery_crystal_1.png",
                "battery_crystal_0.png");
    }
}

File: src/main/kotlin/turniplabs/industry/items/ItemBatteryCrystal.kt
package turniplabs.industry.items

import turniplabs.industry.IndustryConfig

class ItemBatteryCrystal(
    i: Int,
) : ItemBatteryBase(
    i,
    16384,
    IndustryConfig.cfg.getInt("Energy Values.highVoltage"),
    IndustryConfig.cfg.getInt("Energy Values.highVoltage"),
    "battery_crystal_4.png",
    "battery_crystal_3.png",
    "battery_crystal_2.png",
    "battery_crystal_1.png",
    "battery_crystal_0.png"
)